inst/app/www/text/newsletters.yml

```diff
@@ -0,0 +1,31 @@
+newsletters:
+  - image_src: "www/images/New Hall Valley photo with certificate.jpg"
+    image_alt: "A photo of volunteers holding a certificate"
+    title: "September 2025"
+    description: "This newsletter outlined new volunteer group involvement, provided our largest set of results of our data so far, as well as how we are taking this project on the road!"
+    url: "www/forms/BRC Newsletter September 2025.pdf"
+  - image_src: "www/images/Ward end training over tray.jpg"
+    image_alt: "Ward end training over tray"
```

Comment:
Is this a photo? -> "A photo of..."

Reply:
I'm still waiting on alternative text from the researcher for this one, so this is just a placeholder for now.